### PR TITLE
MySQL engine tested and working

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+os:
+  - linux
+language: go
+go:
+  - 1.5
```
Comment (jacobhaven) [inline .travis.yml]: `1.6`

File: schema.go [hunk 2]
```diff
@@ -1,67 +1,63 @@
...
-func QlSchema() string {
-	lines := strings.Split(schema, "\n")
-	var out []string
-	for _, line := range lines {
-		if !strings.HasPrefix(strings.TrimSpace(line), "FOREIGN KEY") {
-			out = append(out, line)
-		}
-	}
-	return strings.Join(out, "\n")
+import ()
```
Comment (jacobhaven) [inline schema.go]: delete

File: sql_test.go [hunk 3]
```diff
@@ -38,7 +41,34 @@ func TestSqliteEngine(t *testing.T) {
...
+	name := os.Getenv("MYSQL_DSN")
+	if name == "" {
+		t.Skip()
+		return
+	}
+
+	// We need to have parseTime=true as one of our DSN paramenters,
+	// so if it's not there, add it on. This way, in sql_scan, we'll get
+	// time.Time values back from MySQL.
+	if !strings.Contains(name, "parseTime=true") {
```
Comment (jacobhaven) [inline sql_test.go]: Why not fully parse the url, set the query parameter, then re-serialize?
```Go
u, err := url.Parse(name)
if err != nil {
	log.Fatal(err)
}
query := u.Query()
query.Set("parseTime", "true")
u.RawQuery = query.Encode()
testEngine(t, "mysql", u.String(), mysqlTimeWriter{})```

File: sql_test.go [hunk 4]
```diff
@@ -38,7 +41,34 @@ func TestSqliteEngine(t *testing.T) {
...
+	name := "./gregor.db"
+	os.Remove(name)
+	testEngine(t, "sqlite3", "./gregor.db", sqliteTimeWriter{})
+}
+
+// Test with: MYSQL_DSN=gregor:@/gregor_test?parseTime=true go test
+func TestMySQLEngine(t *testing.T) {
+	name := os.Getenv("MYSQL_DSN")
+	if name == "" {
+		t.Skip()
```
Comment (jacobhaven) [inline sql_test.go]: Just `t.Skip("MYSQL_DSN not set")`. (No return necessary)

File: sql_test.go (hunk 3 above)
Comment (maxtaco) [inline sql_test.go]: :+1:

File: sql_test.go (hunk 4 above)
Comment (maxtaco) [inline sql_test.go]: :+1:

**Comment by maxtaco** [comment]
Thanks for the review!

**Pull request merged**